go-controller/pkg/ovn/gateway.go

```diff
@@ -385,6 +398,14 @@ func (gw *GatewayManager) GatewayInit(
 	for _, gwLRPIfAddr := range gwLRPIfAddrs {
 		gwLRPNetworks = append(gwLRPNetworks, gwLRPIfAddr.String())
 	}
+	if gw.netInfo.TopologyType() == types.Layer2Topology {
+		// At layer2 GR LRP acts as the layer3 ovn_cluster_router so we need
+		// to configure here the .1 address, this will work only for IC with
+		// one node per zone, since ARPs for .1 will not go beyond local switch.
+		for _, subnet := range hostSubnets {
```

Comment:
in hindsight, having named this attribute host subnets makes this more confusing that it needs to. 

IMHO, we should eventually rename it from host subnets to userDefinedSubnets or something similar.

Reply:
yeah +1 for l2 hostsubnets makes it very confusing